example_crates/drive_example/src/drive_v3_types.rs

```diff
@@ -1,6387 +0,0 @@
-#![allow(unused_variables, unused_mut, dead_code)]
```

Comment:
Why deleted? The point is to serve as an example, the idea is that users of the project generate their own code and can see here how the generated code looks and works. I'd prefer if first-time viewers wouldn't have to first get the generator running just to see whether this is right for them in the first place.

Reply:
We can have those generated codes in the `src/gen` as sub modules but those add unwanted files/mods for others that don't need them. I've explained how to use examples completely in `example/*/README.md` files respectively. They just need to copy paste the shell script...